client/src/scripts/managers/inputManager.ts

```diff
@@ -342,6 +343,50 @@ export class InputManager {
                 shootOnRelease = false;
             });
         }
+        const ticker = new PIXI.Ticker();
+        ticker.stop();
+        ticker.add(() => {
+            const gamepads = navigator.getGamepads();
+            if (gamepads[0]) {
```

Comment:
wouldn't `if (!gamepads[0]) return` be better as it will make the code more readable by removing one level of indentation?